### PR TITLE
Waiting for atleast 2 mongo pods to come up after scaling down to 0 and scaling backup to 3 again

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
As of now , we are waiting for min 1 mongo pod to come up, and then verifying restore status
We need to wait for atleast 2 mongo pod to come up to meet the quorum.
**Which issue(s) this PR fixes** (optional)
Closes #https://purestorage.atlassian.net/browse/PB-7163

**Special notes for your reviewer**:

### CODE REVIEW

**Review by ak-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

File: tests/backup/backup_resiliency_test.go [hunk 1]
```diff
@@ -949,21 +949,21 @@ var _ = Describe("{ScaleMongoDBWhileBackupAndRestore}", Label(TestCaseLabelsMap[
...
-			log.Infof("Verify that at least one mongodb pod is in Ready state")
+			log.Infof("Verify that at least two mongodb pod is in Ready state")
 			mongoDBPodStatus := func() (interface{}, bool, error) {
 				statefulSet, err = apps.Instance().GetStatefulSet(MongodbStatefulset, pxBackupNS)
 				if err != nil {
 					return "", true, err
 				}
 				if statefulSet.Status.ReadyReplicas < 2 {
-					return "", true, fmt.Errorf("no mongodb pods are ready yet")
+					return "", true, fmt.Errorf("minimum 2 mongodb pods are not ready yet")
```
Comment (mkoppal-px) [inline tests/backup/backup_resiliency_test.go:959]: Better to print the number of pods that are ready. That way we will know if the ready pods were `0` or `1`

**Comment by mkoppal-px** [comment]
Is this change tested?

**Review by mkoppal-px — APPROVED** [review]
LGTM

File: tests/backup/backup_resiliency_test.go (hunk 1 above)
Comment (snigdha-px) [reply tests/backup/backup_resiliency_test.go:960]: done

**Comment by snigdha-px** [comment]
> Is this change tested?

https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/5235/consoleFull

**Review by sn-px — APPROVED** [review]
LGTM

**Pull request merged**